lib/engine/game/g_1880/share_pool.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require_relative '../../share_pool'
+
+module Engine
+  module Game
+    module G1880
+      class SharePool < Engine::SharePool
+        def sell_shares(bundle, allow_president_change: true, swap: nil, silent: nil)
```

Comment:
Don't duplicate this method. Add a hook to choose where the share goes. Maybe a new game constant `SOLD_SHARES_DESTINATION = :bank ` and for 80 `:corporation`